### DISCUSSION TITLE
Spring boot 3.0.1

### DISCUSSION THREAD

**Discussion by wesleyzup** [open-telemetry/opentelemetry-java-instrumentation#7689] [Q&A]
Hello.
I'm trying to make use of  the dependency **opentelemetry-spring-boot-starter**, but I noticed that the library only support spring boot 2.7.5.
The problem is that the most classes of the package io.opentelemetry.instrumentation.spring.autoconfigure make use of dependencies from javax.servlet. Although, the new version 3.0.1 of spring boot make use of jarkata.servlet. For this reason,  I received the exception ClassCastException. Can you give-me some alternative to go ahead and fix my problem?

Thank you

**Comment by mateuszrzeszutek** [comment]
Hey @wesleyzup ,

There is an open issue for that upgrade: #7312
